src/plugins/condition/ConditionSetTelemetryProvider.js

```diff
@@ -9,21 +9,33 @@ export default class ConditionSetTelemetryProvider {
         return domainObject.type === 'conditionSet';
     }
 
-    supportsRequest(domainObject, options) {
-        return false;
+    supportsRequest(domainObject) {
+        return domainObject.type === 'conditionSet';
     }
 
     supportsSubscribe(domainObject) {
         return domainObject.type === 'conditionSet';
     }
 
+    request(domainObject, options) {
+        let conditionManager = new ConditionManager(domainObject, this.openmct);
+
+        return conditionManager.requestLADConditionSetOutput()
+            .then(latestOutput => {
+                conditionManager.destroy();
+                conditionManager = undefined;
+                return latestOutput ? [latestOutput] : [];
+            });
+    }
+
     subscribe(domainObject, callback) {
         let conditionManager = new ConditionManager(domainObject, this.openmct);
         conditionManager.on('conditionSetResultUpdated', callback);
 
-        return function unsubscribe() {
+        return () => { // unsubscribe function
```

Comment:
I kind of like how you had this before - calling this `unsubscribe` by name. Makes it clearer.

Reply:
sure. changed back.